strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java

```diff
@@ -42,6 +42,22 @@
     @Inject
     ArtifactEntryService artifactEntryService;
 
+
+
+    @Test
+    public void saveEntityShouldWork()
```

Comment:
Could you please write one more test confirming that created date didn't change on update ?